test/TestStorage.ts

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios';
+import 'mocha';
+import { assert } from 'chai';
+
+import { startAPI } from '../src/api';
+
+describe('Storage tests', async () => {
+    let apiServer;
+
+    before(async () => {
+        apiServer = await startAPI();
+    });
+
+    after(() => {
+        apiServer.close();
+    });
+
+    it('init Origin contract', async () => {
+        const result = await axios.put(
+            'http://localhost:3030/OriginContractLookupMarketLookupMapping/0xb4ec89404c4a24f4c80d157ba9ad803cbc4db614',
+            {
+                marketContractLookup: '0x665b25e0edc2d9b5dee75c5f652f92f5b58be12b'
+            }
+        );
+
+        assert.equal(result.status, 200);
+    });
+
+    it('gets all Origin contracts', async () => {
```

Comment:
Although it's great you've written tests, in my humble opinion these two tests are in fact one test. Second test depends on results of `init Origin contract` test. Please consider having these two tests under one test, because the tests cases are dependent on each other (which is not the best practice), or reset state after first test and initialize it again in second. Thanks.